collections/business/general-notifications/event-general-notifications/account-statement-ready/account-statement-ready/in-app-notification/in-app_channel-settings.json

```diff
@@ -1,5 +1,5 @@
 {
+  "$schema": "../../../../../../../schema/v1/in-app-channel-settings.json",
   "level": "SUCCESS",
-  "targetGroup": "USER",
```

Comment:
This is the same for all definitions, this type does not exist in the schema that EO uses so it is thrown away.

Reply:
You mean that is not a breaking change then, right? We also need to remove from ModelBank repo and wondering whether we have it on Community or not documented somehow, @ajitavbb do you know?

Reply:
Ill put it back and just mark is as deprecated